origin-growth/src/util/locationInfo.js

```diff
@@ -0,0 +1,269 @@
+const restrictedCountryCodes = ['US', 'CA']
+const forbiddenCountryCodes = []
+
+const countryMap = {
+  AD: 'Andorra',
```

Comment:
We'll need to figure out how to localize those strings...
Maybe add a TODO ?

Reply:
I see there are some public repositories of country names in each of the languages. Adding a TODO. 

You think we should have this implemented for initial launch of growth engine?

Reply:
I think it is ok to punt on this for the launch and come back to it a bit later....